packages/z3-static/z3-static.4.8.13/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "simon@aestheticintegration.com"
+authors: "MSR"
+homepage: "https://github.com/Z3prover/z3"
+bug-reports: "https://github.com/Z3prover/z3/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Z3prover/z3.git"
+build: [
+  [ "python2.7" "scripts/mk_make.py" "--ml" "--staticlib" ]
```

Comment:
Could this be changed to python3 (and `conf-python-2-7` to `conf-python-3` below)?

Reply:
I think it would be fine.

I test this update locally (my opam [change](https://github.com/arbipher/opam-repository/commit/58dbab6ccf0ecfb446d20326adecf5fd6a8a3c4a)) and redo all my previous tests. All pass.